blockchain/chain_sync/src/sync.rs

```diff
@@ -247,7 +246,7 @@ where
     /// Syncs messages by first checking state for message existence otherwise fetches messages from blocksync
     async fn sync_messages_check_state(&mut self, ts: &[Tipset]) -> Result<(), Error> {
         // see https://github.com/filecoin-project/lotus/blob/master/build/params_shared.go#L109 for request window size
-        const REQUEST_WINDOW: i64 = 200;
+        const REQUEST_WINDOW: i64 = 1;
```

Comment:
Nit: Add TODO as a reminder to change this back to 200 as its the `defaultMessageFetchWindowSize` in Lotus.

Reply:
intentionally not followed for now, doesn't make sense why this request would be bulked like this since the transition has to be done sequentially. Can benchmark on our own client if batching the requests helps throughput, but this is not a protocol value